packages/updatePrescriptionStatus/src/utils/databaseClient.ts

```diff
@@ -0,0 +1,35 @@
+import {Logger} from "@aws-lambda-powertools/logger"
+import {DynamoDBClient, TransactWriteItem, TransactWriteItemsCommand} from "@aws-sdk/client-dynamodb"
+import {marshall} from "@aws-sdk/util-dynamodb"
+
+import {DataItem} from "../updatePrescriptionStatus"
+
+const logger = new Logger({serviceName: "databaseClient"})
+const client = new DynamoDBClient({region: "eu-west-2"})
```

Comment:
do we need to specify the region, fairly sure it's not going to change but it seems like asking for trouble to hard code it

Reply:
Yeah, good point. No longer hard-coded.